### DISCUSSION TITLE
Restored webview background color is hardcoded to white

### DISCUSSION THREAD

**Discussion by futurepaul** [capawesome-team/capacitor-mlkit#92] [Q&A] [answered]
https://github.com/capawesome-team/capacitor-mlkit/blob/fb8f999da1b83452e62ef1fb2f48ef5051f66cc8/packages/barcode-scanning/ios/Plugin/BarcodeScanner.swift#L225

It's possible with Capacitor to set the webview's background color. Is there a way to recall that color when restoring the webview background?

**Comment by robingenz** [comment, marked as answer]
Yes, it should be possible to read the current value of `webView.backgroundColor`.
Do you have any specific problems?
